synapse/replication/tcp/handler.py

```diff
@@ -567,6 +567,11 @@ async def _process_position(
         # between then and now.
         missing_updates = cmd.prev_token != current_token
         while missing_updates:
+            # Note: There may very well not be any new updates, but we check to
+            # make sure. This can particularly happen for the event stream where
+            # event persisters continuously send `POSITION`. See `resource.py`
+            # for why this can happen.
```

Comment:
Just for my own curiosity/understanding, is this the comment in `resource.py` that you are referring to? https://github.com/matrix-org/synapse/blob/9ce51a47f6e37abd0a1275281806399d874eb026/synapse/replication/tcp/resource.py#L100

Reply:
Yeah, it's that one basically.